js/test/test.js

```diff
@@ -306,6 +306,7 @@ async function testExchange (exchange) {
     await test ('fetchClosedOrders', exchange, symbol)
     await test ('fetchMyTrades', exchange, symbol)
     await test ('fetchLeverageTiers', exchange, symbol)
+    await test ('fetchOpenInterestHistory', exchange, symbol)
```

Comment:
Is it a public method or a private method in most cases?

Reply:
public

Reply:
If it's public, then we should test it in the public section, not in the private section, right?